plugin/storage/cassandra/schema.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```

Comment:
looks ok here, given the other scripts. I would just rename it to `create-schema.sh`